src/python/zquantum/core/interfaces/backend.py

```diff
@@ -19,7 +18,7 @@
 
 # Note that in particular Wavefunction is a StateVector. However, for performance
 # reasons QuantumSimulator uses numpy arrays internally.
-StateVector = Union[Sequence[Complex], np.ndarray]
+StateVector = Union[Sequence[complex], np.ndarray]
```

Comment:
[suggestion] Perhaps move this to `zquantum.core.typing`?

Reply:
Konrad and I  have that planned for a different story.

Reply:
It's a different story because if we do that then we need to update the placement of `StateVector` in all the other repos.